src/main/java/com/bobocode/hoverla/bring/context/BeanInitializer.java

```diff
@@ -7,7 +7,13 @@ public class BeanInitializer {
     /**
      * Instantiates beans and injects dependencies in them
      */
-    void initialize(Map<String, BeanDefinition> beanDefinitions) {
-
+    public void initialize(Map<String, BeanDefinition> beanDefinitions) {
+        for (Map.Entry<String, BeanDefinition> entry : beanDefinitions.entrySet()) {
+            BeanDefinition beanDefinition = entry.getValue();
+            BeanDefinition[] instanceDependencies = beanDefinition.dependenciesNames().stream()
```

Comment:
Can we align method chains like this below, and also update code style xml and push it in this PR?
```
beanDefinition.dependenciesNames()
              .stream()
              .map(beanDefinitions::get)
              .toArray(BeanDefinition[]::new);
```